pkg/cmd/server/kubernetes/node/node_config.go

```diff
@@ -374,9 +375,18 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig, enableProxy, enable
 	return config, nil
 }
 
-func buildKubeProxyConfig(options configapi.NodeConfig) (*proxyoptions.ProxyServerConfig, error) {
+func buildKubeProxyConfig(options configapi.NodeConfig) (*componentconfig.KubeProxyConfiguration, error) {
+	proxyOptions, err := kubeproxyoptions.NewOptions()
+	if err != nil {
+		return nil, err
+	}
+	// Resolve cmd flags to add any user overrides
+	if err := cmdflags.Resolve(options.ProxyArguments, proxyOptions.AddFlags); len(err) > 0 {
```

Comment:
Eventually we want to enter with the config, not the options.  Not all the overrides being expressed will have args.  I'll see if I can bring the remaining ones back into options.